docs/elasticsearch-specification.asciidoc

```diff
@@ -264,6 +265,32 @@ $ openssl req -x509 -sha256 -nodes -newkey rsa:4096 -days 365 -subj "/CN=quickst
 $ kubectl create secret generic my-cert --from-file=ca.crt=tls.crt --from-file=tls.crt=tls.crt --from-file=tls.key=tls.key
 ----
 
+
+[id="{p}-reserved-settings"]
+=== Settings managed by ECK
+
+The following Elasticsearch settings are managed by ECK:
+
+* `cluster.name`
+* `discovery.zen.minimum_master_nodes` deprecated[7.0]
+* `cluster.initial_master_nodes` added[7.0]
+* `network.host`
+* `network.publish_host`
+* `path.data`
+* `path.logs`
+* `xpack.security.authc.reserved_realm.enabled`
+* `xpack.security.enabled`
+* `xpack.security.http.ssl.certificate`
+* `xpack.security.http.ssl.enabled`
+* `xpack.security.http.ssl.key`
+* `xpack.security.transport.ssl.certificate`
+* `xpack.security.transport.ssl.enabled`
+* `xpack.security.transport.ssl.key`
+* `xpack.security.transport.ssl.verification_mode`
+
+CAUTION: While ECK does not prevent you from setting any of these settings yourself, you are strongly discouraged from doing so and any user provided Elasticsearch configuration that does use one of these settings is unsupported.
```

Comment:
1. Maybe: s/that does use one of these/that does use any of these

2. 
> is unsupported

Maybe it's just me, but this could sound like it just doesn't work (while it might, be we are not providing support for it).

Reply:
I am open to suggestions how to rephrase it

Reply:
```
While ECK does not prevent you from setting any of these settings yourself, you are strongly discouraged from doing so and we cannot offer support for any user provided Elasticsearch configuration that does use any of these settings.
```
Better?

Reply:
Yes, thanks.